code/DDSCodeTester.cpp

```diff
@@ -988,7 +988,7 @@ void dds_domain_examples()
 
         pqos.properties().properties().emplace_back(
             "fastdds.serialize_optional_qos",
-            "True");
+            "true"); // True or TRUE or 1
```

Comment:
```suggestion
            "true"); // true or True or TRUE or 1
```
Even if it looks obvious, I would go for a redundant approach to avoid any sort of potential doubts. It will also apply to the XML example